app/models/ability.rb

```diff
@@ -65,13 +65,18 @@ def initialize(user)
 
     # Events
     can [:read, :create], Event
+    can [:participants], Event do |r|
+      user.organizing_events.exists?(r.id)
```

Comment:
Yes, it is working. r is for event actually we are using r for many other names so used here also, changing it :)